api/src/test/java/gov/cms/ab2d/api/controller/AuthenticationTests.java

```diff
@@ -0,0 +1,108 @@
+package gov.cms.ab2d.api.controller;
+
+import com.okta.jwt.JwtVerificationException;
+import gov.cms.ab2d.api.SpringBootApp;
+import gov.cms.ab2d.common.model.User;
+import gov.cms.ab2d.common.repository.JobRepository;
+import gov.cms.ab2d.common.repository.RoleRepository;
+import gov.cms.ab2d.common.repository.SponsorRepository;
+import gov.cms.ab2d.common.repository.UserRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.List;
+
+import static gov.cms.ab2d.api.controller.TestUtil.TEST_USER;
+import static gov.cms.ab2d.api.util.Constants.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = SpringBootApp.class, webEnvironment = SpringBootTest.WebEnvironment.MOCK)
+@AutoConfigureMockMvc
+public class AuthenticationTests {
```

Comment:
We are missing a test case that covers `JwtVerificationException`, i.e. when a token fails validation. So I guess `TestUtils` needs a `setupInvalidToken` method that will configure the verifier mock to throw `JwtVerificationException`.